src/main/java/com/st/utopia/cancellationservice/model/Ticket.java

```diff
@@ -0,0 +1,163 @@
+package com.st.utopia.cancellationservice.model;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * A Ticket model.
+ *
+ * 
+ * Model used for the purpose of the cancellation microservice.
+ * 
+ * There are no set methods for information about the ticket such as flight
+ * number, row/seat, or class.
+ * 
+ * Upon cancelling the ticket, the microservice needs to set the following
+ * fields as null: -reserver -price -reservation_timeout -booking_id
+ * 
+ * This model does not need to define a composite key since the cancellation
+ * microservice will search for the ticket via booking_id which is unique
+ * 
+ * 
+ * @author Al-amine AHMED MOUSSA
+ */
+
+@Entity
+@Table(name = "tbl_tickets")
+public class Ticket {
+	/**
+	 * The three fields that together constitute the identity of a Ticket.
+	 */
+	@EmbeddedId
+	private TicketIdentity id;
+
+	@Column(name = "class")
+	private Integer tier; // class is a keyword
+
+	private Integer reserver;
+
+	private Integer price;
+
+	private LocalDateTime reservationTimeout;
+
+	private String bookingId;
+
+	/**
+	 * the Getters and Setters of the Class
+	 */
+	public TicketIdentity getId() {
+		return id;
+	}
+
+	public Integer getFlight() {
+		if (id == null) {
+			return null;
+		} else {
+			return id.getFlight();
+		}
+	}
+
+	public Integer getRow() {
+		if (id == null) {
+			return null;
+		} else {
+			return id.getRow();
+		}
+	}
+
+	public char getSeat() {
+		if (id == null) {
+			return 0;
+		} else {
+			return id.getSeat();
+		}
+	}
+
+	public Integer getTier() {
+		return tier;
+	}
+
+	public void setTier(Integer tier) {
+		this.tier = tier;
+	}
+
+	public Integer getReserver() {
+		return reserver;
+	}
+
+	public void setReserver(Integer reserver) {
+		this.reserver = reserver;
+	}
+
+	public Integer getPrice() {
+		return price;
+	}
+
+	public void setPrice(Integer price) {
+		this.price = price;
+	}
+
+	public LocalDateTime getReservationtimeout() {
```

Comment:
Getters and setters should have the same capitalization (except for capitalizing the letter after `get`, `set`, or `is`) as the field they access; in method names you consistently say `reservationtimeout` and `bookingid` when the fields are `reservationTimeout` and `bookingId`.